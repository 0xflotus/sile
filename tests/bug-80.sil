\begin[papersize=a5]{document}

\define[command=item]{\quad â€“ \process}

Primum, inquam, deprecor, ne me tamquam philosophum putetis scholam vobis aliquam explicaturum, quod ne in ipsis quidem philosophis magnopere umquam probavi.

 \item{Quando enim Socrates, qui parens philosophiae iure dici potest, quicquam tale fecit?}

\end{document}
