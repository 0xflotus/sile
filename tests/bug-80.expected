Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.9764
My 	37.3321
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 85 76 80 88 80	(Primum)
Mx 	74.2381
T	15	(,)
Mx 	80.0371
T	76 81 84 88 68 80	(inquam)
Mx 	111.3359
T	15	(,)
Mx 	117.1349
T	71 72 83 85 72 70 82 85	(deprecor)
Mx 	154.1906
T	15	(,)
Mx 	159.9895
T	81 72	(ne)
Mx 	173.6352
T	80 72	(me)
Mx 	189.7955
T	87 68 80 84 88 68 80	(tamquam)
Mx 	232.4402
T	83 75 76 79 82 86 82 83 75 88 80	(philosophum)
Mx 	290.2509
T	83 88 87 72 87 76 86	(putetis)
Mx 	322.4317
T	86 70 75 82 79 68 80	(scholam)
Mx 	360.0520
T	89 82 69 76 86	(vobis)
Mx 	385.1723
T	68 79 76	(ali)
Mx 	395.1820
T	16	(-)
Mx 	20.9764
My 	49.3321
T	84 88 68 80	(quam)
Mx 	46.6922
T	72 91 83 79 76 70 68 87 88 85 88 80	(explicaturum)
Mx 	102.0486
T	15	(,)
Mx 	106.9832
T	84 88 82 71	(quod)
Mx 	130.3064
T	81 72	(ne)
Mx 	143.0876
T	76 81	(in)
Mx 	153.9597
T	76 83 86 76 86	(ipsis)
Mx 	175.0466
T	84 88 76 71 72 80	(quidem)
Mx 	208.7018
T	83 75 76 79 82 86 82 83 75 76 86	(philosophis)
Mx 	258.8903
T	80 68 74 81 82 83 72 85 72	(magnopere)
Mx 	308.0777
T	88 80 84 88 68 80	(umquam)
Mx 	347.1236
T	83 85 82 69 68 89 76	(probavi)
Mx 	378.7203
T	17	(.)
Mx 	54.6451
My 	61.3321
T	178	(â€“)
Mx 	63.0063
T	52 88 68 81 71 82	(Quando)
Mx 	98.3108
T	72 81 76 80	(enim)
Mx 	122.8584
T	54 82 70 85 68 87 72 86	(Socrates)
Mx 	157.4922
T	15	(,)
Mx 	163.4510
T	84 88 76	(qui)
Mx 	180.2789
T	83 68 85 72 81 86	(parens)
Mx 	211.7943
T	83 75 76 79 82 86 82 83 75 76 68 72	(philosophiae)
Mx 	268.3537
T	76 88 85 72	(iure)
Mx 	288.6094
T	71 76 70 76	(dici)
Mx 	307.2684
T	83 82 87 72 86 87	(potest)
Mx 	332.9617
T	15	(,)
Mx 	338.9205
T	84 88 76 70 84 88 68 80	(quicquam)
Mx 	383.1899
T	87 68 79 72	(tale)
Mx 	20.9764
My 	73.3321
T	73 72 70 76 87	(fecit)
Mx 	39.2283
T	34	(?)
Mx 	207.4176
My 	553.7327
T	20	(1)
End page
Finish
